backend/README.md

```diff
@@ -38,7 +38,50 @@ The above versions are what have been used for development, however, other versi
 
 ---
 
-## Setup
+## Requirements to Run Tests
+
+To run the unit tests, you will need Docker installed.  
```

Comment:
do you want to add the command needed to check if your computer has Docker already or not?
- might help for new devs?